pkg/image/registry/imagestreamtag/rest.go

```diff
@@ -315,34 +315,47 @@ func (r *REST) Delete(ctx apirequest.Context, id string, options *metav1.DeleteO
 		return nil, false, err
 	}
 
-	stream, err := r.imageStreamRegistry.GetImageStream(ctx, name, &metav1.GetOptions{})
-	if err != nil {
-		return nil, false, err
-	}
+	for i := 10; i > 0; i-- {
```

Comment:
Out of curiosity: why exponential backoff isn't used here?

Reply:
it's a conflict, the assumption is we should be able to fetch the new/updated object and try again immediately.  backing off wouldn't provide any value in that scenario unless we think there is a storm of updates occurring, in which case we'd want a random backoff, not an exponential one.